shopping/app/src/main/res/values/strings.xml

```diff
@@ -6,4 +6,6 @@
     <string name="playlist_id">Playlist ID:</string>
 
     <string name="checkout">Checkout</string>
+    <string name="shop_now">Shop now mode</string>
+    <string name="add_to_cart">Add to cart mode</string>
```

Comment:
I personally won't mention mode, but not a deal breaker
```suggestion
    <string name="shop_now">Shop now</string>
    <string name="add_to_cart">Add to cart</string>
```

Reply:
Was not sure about this as well